corehq/apps/app_manager/templates/app_manager/v1/summary.html

```diff
@@ -36,6 +36,7 @@
                 $httpProvider.defaults.headers.common['X-Requested-With'] = 'XMLHttpRequest';
                 $httpProvider.defaults.xsrfCookieName = 'csrftoken';
                 $httpProvider.defaults.xsrfHeaderName = 'X-CSRFToken';
+                $httpProvider.defaults.headers.common["X-CSRFToken"] = '{{ csrf_token }}';
```

Comment:
That sounds right (merging master) - apologies for not having gotten to look at this, but glad you figured it out.